<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <node pkg="eagleye_fix2pose" name="fix2pose_node" exec="fix2pose" output="screen" >
    <!-- /eagleye/fix or  /eagleye/rtk_fix(rtk_dead_reckoning mode) or /GNSS/fix(GNSS pose only mode)-->
    <remap from="eagleye/fix" to="eagleye/fix" />
    <remap from="ekf_trigger_node" to="/localization/pose_twist_fusion_filter/trigger_node"/>

    <!-- plane rectangular coordinate number -->
    <param name="plane" value="7"/>
    <!-- 1 : plane rectangular coordinate  2 : MGRS -->
    <param name="tf_num" value="1"/>
    <!-- 0 : No convert  1 : ellipsoid -> altitude  2 : altitude -> ellipsoid -->
    <param name="convert_height_num" value="0"/>
    <!-- 0 : EGM2008-1  1 : GSIGE2011 Ver2.1 -->
    <param name="geoid_type" value="0"/>

    <param name="parent_frame_id" value="map"/>
    <param name="child_frame_id" value="eagleye_base_link"/>
    <param name="base_link_frame_id" value="base_link"/>
    <param name="gnss_frame_id" value="gnss_link"/>

    <param name="fix_only_publish" value="false"/>
    <!-- 0:fix.status 1:fix.position_covariance[0] (if fix_only_publish==true)-->
    <param name="fix_judgement_type" value="0"/>
    <!--Fix status(if fix_judgement_type is 0)-->
    <param name="fix_gnss_status" value="0"/>
    <!-- Minimum covariance of the pose topics to be published(if fix_judgement_type is 0)-->
    <param name="fix_std_pos_thres" value="0.1"/>


  </node>

</launch>
